config.json

```diff
@@ -25,7 +25,6 @@
         "duration": 1200,
         "lifetime": 86400,
         "taskUrl": "",
-        "production": false,
```

Comment:
Could you please add this back? See comment above. 

Used here: https://github.com/a-newman/mturk-template/blob/master/assets/js/main.js#L144